cli/main.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+from cli.app import CONF, IRFactory
+
+
+def main_provision():
```

Comment:
These dedicated methods are used as entry points for ir-provisioner/ir-installer scripts . We cannot have entry point with parameters. So somehow we need to have different methods in one module or methods in different modules.